pkg/raid/raid.go

```diff
@@ -166,7 +166,8 @@ func assembleExistingMirror(target, existing string, devices ...string) error {
 }
 
 func createNewStriped(target string, devices ...string) error {
-	output, err := runMdadm(slices.Concat([]string{"--create", target, "--level", "0", "--run", "--raid-devices", fmt.Sprintf("%d", len(devices))}, devices)...)
+	// Force is needed if the number of devices is 1.
```

Comment:
Should this comment be "if devices >= 1"?

I'm also assuming that devices has at least one element (and this command wouldn't run if len(devices) == 0